<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <item>
       <widget class="QTextBrowser" name="text_inf">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QVTKOpenGLWidget" name="qvtkWidgetLeft">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="action_input"/>
    <addaction name="action_exportInf"/>
    <addaction name="action_exportStl"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>view</string>
    </property>
    <addaction name="action_anatomyView"/>
    <addaction name="action_3dView"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>settings</string>
    </property>
    <addaction name="action_buildparm"/>
    <addaction name="action_language"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="action_thisproject"/>
   </widget>
   <addaction name="file"/>
   <addaction name="about"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_close">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="DividingCubes">
   <property name="text">
    <string>剖分立方体算法</string>
   </property>
  </action>
  <action name="RayCasting">
   <property name="text">
    <string>光线投射算法</string>
   </property>
  </action>
  <action name="TextureMapping">
   <property name="text">
    <string>3D纹理映射算法</string>
   </property>
  </action>
  <action name="MaximumIntensity">
   <property name="text">
    <string>最大密度投影算法</string>
   </property>
  </action>
  <action name="action_exportInf">
   <property name="text">
    <string>print tags</string>
   </property>
  </action>
  <action name="action_anatomyView">
   <property name="text">
    <string>slice view</string>
   </property>
  </action>
  <action name="action_input">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="MarchingCubes">
   <property name="text">
    <string>移动立方体算法</string>
   </property>
  </action>
  <action name="action_exportSTL">
   <property name="text">
    <string>导出STL</string>
   </property>
  </action>
  <action name="action_3dView">
   <property name="text">
    <string>3D view</string>
   </property>
  </action>
  <action name="action_buildparm">
   <property name="text">
    <string>parameter</string>
   </property>
  </action>
  <action name="action_language">
   <property name="text">
    <string>language</string>
   </property>
  </action>
  <action name="action_thisproject">
   <property name="text">
    <string>about us</string>
   </property>
  </action>
  <action name="action_exportStl">
   <property name="text">
    <string>output to STL</string>
   </property>
  </action>
  <action name="action_test">
   <property name="text">
    <string>test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
